GSA/digital-gov-drupal
# DIGITAL-141 Rabbit Hole module

## Jira ticket




[DIGITAL-141](https://cm-jira.usa.gov/browse/DIGITAL-141)

## Purpose

Installs and enables the rabbit hole module.
Enables the rg_node submodule
Configures the short post, source, and guide nav content types to 404 when viewed



## Includes the following PRs that must be merged first

[DIGITAL-47](https://github.com/GSA/digital-gov-drupal/pull/82) which adds the guide nav content type.



## Deployment and testing

### Local Setup



`lando si` 

### QA/Testing instructions

Visit the short post, source, and guide nav nodes and confirm they show a 404 message



## Checklist for the Developer




## Checklist for the Peer Reviewers
